web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/IiifV3ManifestControllerTest.java

```diff
@@ -141,4 +144,72 @@ public void testGetCanvas() throws Exception {
         var items = (List) respMap.get("items");
         assertFalse(items.isEmpty());
     }
+
+    @Test
+    public void testGetManifestWithAVFiles() throws Exception {
+        var workObj = new ContentObjectSolrRecord();
+        workObj.setId(OBJECT_ID);
+        workObj.setResourceType(ResourceType.Work.name());
+        workObj.setTitle("Test Work");
+        when(accessCopiesService.listViewableFiles(eq(OBJECT_PID), any())).thenReturn(Arrays.asList(workObj));
+
+        var result = mockMvc.perform(get("/iiif/v3/" + OBJECT_ID + "/manifest")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn();
+
+        Map<String, Object> respMap = MvcTestHelpers.getMapFromResponse(result);
+        assertEquals("Manifest", respMap.get("type"));
+        assertEquals("http://example.com/iiif/v3/f277bb38-272c-471c-a28a-9887a1328a1f/manifest", respMap.get("id"));
+        assertEquals("Test Work", ((List) ((Map) respMap.get("label")).get("none")).get(0));
+        var metadata = (List) respMap.get("metadata");
+        assertFalse(metadata.isEmpty());
+    }
+
+    @Test
+    public void testGetCanvasWithVideoFile() throws Exception {
+        var fileObj = new ContentObjectSolrRecord();
+        fileObj.setId(OBJECT_ID);
+        fileObj.setResourceType(ResourceType.File.name());
+        fileObj.setTitle("File Object");
+        var originalDs = new DatastreamImpl("original_file|video/mp4|video.mp4|mp4|0|||240x750x500");
+        fileObj.setDatastream(List.of(originalDs.toString()));
+        when(accessCopiesService.listViewableFiles(eq(OBJECT_PID), any())).thenReturn(List.of(fileObj));
+
+        var result = mockMvc.perform(get("/iiif/v3/" + OBJECT_ID + "/canvas")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn();
+
+        var respJson = MvcTestHelpers.getResponseAsJson(result);
+        var body = respJson.get("items").get(0).get("items").get(0).get("body");
+        assertEquals(SERVICES_BASE + "file/" + OBJECT_ID, body.get("id").textValue());
```

Comment:
Just a suggestion, but it would be good to use a consistent format for how the expected URI is supplied, since sometimes its by adding together constants and sometimes its a full string. I might have a slight preference towards listing the whole string each time since its easier to read, but either is fine.

Reply:
Okay, I'll make it consistent in the whole test!